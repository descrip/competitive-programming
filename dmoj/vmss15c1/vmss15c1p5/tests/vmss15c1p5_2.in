9 927 978
apple 97 27 4
fertig 306 22 12
gala 533 24 33
laptop 763 26 33
morris 702 14 5
mountain 986 30 10
reddit 231 3 6
stang 428 17 11
yokoono 656 5 25
