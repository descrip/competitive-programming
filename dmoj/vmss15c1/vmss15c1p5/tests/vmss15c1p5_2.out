57607
apple 0
fertig 0
gala 0
laptop 0
morris 53
mountain 0
reddit 23
stang 0
yokoono 23
